### DISCUSSION TITLE
3D UNet Model for Predicting 2D DICOM Slices?

### DISCUSSION THREAD

**Discussion by sepidhk** [Project-MONAI/MONAI#8250] [General]
Hi All,

I have trained a 3D UNet model on 3D MRI volumes (Nifti) with dimensions (width: 512, height: 512, slices: 64). Now, I have a set of 2D DICOM slices (not a 3D volume). Is it possible to use the 3D-trained model to predict the 2D slices? Any thoughts?

Thank you in advance for your help.

**Comment by KumoLiu** [comment]
Hi @sepidhk, using a 3D U-Net model trained on 3D MRI volumes to predict 2D DICOM slices is possible but not straightforward due to differences in input dimensionality.
You may:
- Modify the Model, such as adapt the model architecture to accept 2D inputs by reducing one dimension of the convolutions (e.g., from 3D Conv to 2D Conv layers) then fine-tune the model to train the adjusted model on a dataset of 2D slices to refine the weights.
- Treat each 2D slice independently, padding or replicating it along the missing dimension to create a pseudo-3D volume, e.g., (512, 512, 1) or (512, 512, 3) then feed these pseudo-volumes into the model for prediction.
